psi4/src/read_options.cc

```diff
@@ -1544,6 +1544,30 @@ int read_options(const std::string &name, Options &options, bool suppress_printi
         /*- Bump function max radius -*/
         options.add_double("DF_BUMP_R1", 0.0);
 
+        /*- SUBSECTION COSX Algorithm -*/
+
+        /*- Number of spherical points in initial COSX grid. -*/
+        options.add_int("COSX_SPHERICAL_POINTS_INITIAL", 50);
+        /*- Number of radial points in initial COSX grid. -*/
+        options.add_int("COSX_RADIAL_POINTS_INITIAL", 25);
+        /*- Number of spherical points in final COSX grid. -*/
+        options.add_int("COSX_SPHERICAL_POINTS_FINAL", 110);
+        /*- Number of radial points in final COSX grid. -*/
+        options.add_int("COSX_RADIAL_POINTS_FINAL", 35);
```

Comment:
These values are pretty similar to the defaults given in the COSX papers. We can't do an apples-to-apples comparison with ORCA, because in ORCA, the number of radial points is a function of the element's row.

I've ran calculations on sulfur-containing systems just fine, and gotten reasonable results.